diff --git a/dist/esm/index.esm2017.js b/dist/esm/index.esm2017.js
index b4c53c4d4271088e715ef7ef37b10b6bfa179638..5a7641a6d5ed1f873fcefb20e49775ac0cfa6996 100644
--- a/dist/esm/index.esm2017.js
+++ b/dist/esm/index.esm2017.js
@@ -562,11 +562,17 @@ async function getNewToken(firebaseDependencies, subscriptionOptions) {
  */
 async function getPushSubscription(swRegistration, vapidKey) {
     const subscription = await swRegistration.pushManager.getSubscription();
-    if (subscription) {
-        return subscription;
+ 
+   if (subscription) {
+        if (!subscription.options.userVisibleOnly) {
+            return subscription;
+        }
+
+        await subscription.unsubscribe()
     }
+
     return swRegistration.pushManager.subscribe({
-        userVisibleOnly: true,
+        userVisibleOnly: false,
         // Chrome <= 75 doesn't support base64-encoded VAPID key. For backward compatibility, VAPID key
         // submitted to pushManager#subscribe must be of type Uint8Array.
         applicationServerKey: base64ToArray(vapidKey)
