diff --git a/styles/createThemeNoVars.js b/styles/createThemeNoVars.js
index 155988969c4947d85fcde9cb1b55faa266cb12f7..0e88684fbc7a61b5c4c726826b6c869da8ce9c59 100644
--- a/styles/createThemeNoVars.js
+++ b/styles/createThemeNoVars.js
@@ -67,7 +67,7 @@ function createThemeNoVars(options = {}, ...args) {
     };
     Object.keys(muiTheme.components).forEach(component => {
       const styleOverrides = muiTheme.components[component].styleOverrides;
-      if (styleOverrides && component.startsWith('Mui')) {
+      if (styleOverrides && typeof styleOverrides !== 'string' && component.startsWith('Mui')) {
         traverse(styleOverrides, component);
       }
     });
