diff --git a/node_modules/jest-runner/build/index.js b/node_modules/jest-runner/build/index.js
index 65c0ed1..3bb4938 100644
--- a/node_modules/jest-runner/build/index.js
+++ b/node_modules/jest-runner/build/index.js
@@ -124,7 +124,7 @@ class TestRunner extends _types.EmittingTestRunner {
       enableWorkerThreads: this._globalConfig.workerThreads,
       exposedMethods: ['worker'],
       forkOptions: {
-        serialization: 'json',
+        serialization: 'advanced',
         stdio: 'pipe'
       },
       // The workerIdleMemoryLimit should've been converted to a number during
